### PR TITLE
Docs/dialog documentation

### PR DESCRIPTION
**Pull request by DanielleRameau**
Added basic dialog documentation

### CODE REVIEW

File: website/src/categories/components/dialog/code.md [hunk 1]
```diff
@@ -7,29 +7,68 @@ eleventyNavigation:
   key: DialogCode
 ---
 <section>
```
Comment (anna-lach) [inline website/src/categories/components/dialog/code.md:9]: Can you please add installation section to the dialog like in the switch PR https://github.com/sl-design-system/components/pull/953

File: website/src/categories/components/dialog/usage.md [hunk 2]
```diff
@@ -5,27 +5,154 @@ eleventyNavigation:
...
+</div>
+  
+</section>
+
+<section>
+  
+## Behavior
+
+### Height
+The height of the dialog is dynamically calculated as a percentage of the available screen. As content is injected into the dialog, it grows vertically to accommodate the information passed to it.
```
Comment (jpzwarte) [inline website/src/categories/components/dialog/usage.md:136]: This reads weird: the height of the dialog is based on the width and the content. It's not "dynamically calculated as a percentage of the available screen"?

File: website/src/categories/components/dialog/usage.md [hunk 3]
```diff
@@ -5,27 +5,154 @@ eleventyNavigation:
...
+
+### Entrance and exit motion
+When the user launches the dialog, the overlay fades in from 0% to 100% opacity (transitions) and the dialog slides upward from the centre of the screen (transforms:translate the x or y axis). When the user dismisses the dialog, the same animation occurs in reverse.
```
Comment (jpzwarte) [inline website/src/categories/components/dialog/usage.md:139]: Should we have these detailed descriptions here of animations? What if we change animations in the future? We might forget to update the docs? And this also doesn't describe how the animation migth/will be different on mobile.

File: website/src/categories/components/dialog/usage.md [hunk 4]
```diff
@@ -5,27 +5,154 @@ eleventyNavigation:
...
+
+### Content overflow
+When content exceeds the available space, it overflows and becomes scrollable.
```
Comment (jpzwarte) [inline website/src/categories/components/dialog/usage.md:142]: *vertical space*.

File: website/src/categories/components/dialog/usage.md [hunk 5]
```diff
@@ -5,27 +5,154 @@ eleventyNavigation:
   parent: Dialog
   key: DialogUsage
 ---
+
+<section>
+<div class="ds-example">
```
Comment (jpzwarte) [inline website/src/categories/components/dialog/usage.md:10]: Perhaps show a less technical example on the Usage tab? (have it work the same, but make the displayed code less complex). Still show this technical example on the Code tab.

File: website/src/categories/components/dialog/accessibility.md [hunk 6]
```diff
@@ -7,32 +7,38 @@ eleventyNavigation:
...
+<div class="ds-table-wrapper">
 
-</section>
-
-<section>
+|Command|Description|
+|-|-|
+|Tab|Moves focus to the next tabbable element inside the dialog. If focus is on the last tabbable element, it wraps back to the first.|
+|Shift + Tab|Moves focus to the previous tabbable element inside the dialog. If focus is on the first, it wraps to the last.|
+|Escape (ESC)|Closes the dialog.|
```
Comment (jpzwarte) [inline website/src/categories/components/dialog/accessibility.md:18]: Shouldn't we mention that this behavior is disabled when `disableCancel` is set?

**Review by jpzwarte — APPROVED** [review]

File: website/src/categories/components/dialog/usage.md (hunk 2 above)
Comment (DanielleRameau) [reply website/src/categories/components/dialog/usage.md:136]: updated text

File: website/src/categories/components/dialog/usage.md (hunk 3 above)
Comment (DanielleRameau) [reply website/src/categories/components/dialog/usage.md:139]: yeah, good one, will remove it for now

**Pull request merged**